Dockerfile

```
@@ -0,0 +1,14 @@
+# 1. JRE 이미지를 베이스로 사용 (실행 전용, JDK보다 경량)
+FROM eclipse-temurin:17-jre-jammy
+
+# 2. 빌드된 JAR 파일 경로를 변수로 정의
+ARG JAR_FILE=build/libs/*.jar
+
+# 3. JAR 파일을 컨테이너 내부로 복사
+COPY ${JAR_FILE} app.jar
```

Hadolint (2.14.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)